Puppetfile

```diff
@@ -10,6 +10,10 @@ mod 'theforeman/dns',           :git => 'https://github.com/theforeman/puppet-dn
 mod 'theforeman/git',           :git => 'https://github.com/theforeman/puppet-git'
 mod 'theforeman/tftp',          :git => 'https://github.com/theforeman/puppet-tftp'
 
+# From git until PassengerMaxInstancesPerApp is available in released version
+#   https://github.com/puppetlabs/puppetlabs-apache/commit/d14e1a83e7e778fc6a000f8b28124fdb36834c43
+mod 'puppetlabs/apache',        :git => 'https://github.com/puppetlabs/puppetlabs-apache.git'
```

Comment:
> If we really so want that one change (and I really would like to have serious prove it's worth it)

With 6 instances of Passenger, busy puppet clients will take up 5, leaving only 1 to respond to Foreman UI requests.  Or in the reverse, a busy Foreman instance with multiple users will leave only 1 to respond to Puppet requests.  It's  a very unpleasant experience.

The evidence that this is needed is the numerous Red Hat customers, Katello users, and even Foreman users who have tried submitting PR's to fix it for years.

You can deal without the max_instances_per_app that I'm trying to get the newer PL-apache change - but you'll only be a little better off.  You'll be giving passenger 12 instances to deal with it, but you just end up with the same problem once you get under load.  You could have Puppet or Foreman using 11 and leaving the other with 1. The mitigation only goes so far.

The max_instances_per_app change is very much needed, it's the whole reason it exists is when you have multiple passenger apps competing for resources.